canonical/ubuntu.com
# Rebrand /download/server pages

## Done

- Applied rebrand as per [doc](https://docs.google.com/document/d/1Qob4Ni_y1ffIdSWqaw2xH0dxiHZQmkRGP5nuoBhIfyM/edit) and [mock-up](https://www.figma.com/file/cVGimTPHky4h3BPXfrT6GB/24.04-U.com---download-(rebranding)?node-id=906%3A7906&mode=dev)

## QA

- View the site locally in your web browser at: https://ubuntu-com-13704.demos.haus/download/server
    - Be sure to test on mobile, tablet and desktop screen sizes
- Check against doc and design
- Please make sure each download still works as expected

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-8497
